deps/secure-socket/biowrap.lua

```diff
@@ -48,6 +48,10 @@ return function (ctx, isServer, socket, handshakeComplete, servername)
 
   local function handshake(callback)
     if ssl:handshake() then
+      local cert = ssl:peer()
```

Comment:
`cert` can be `nil`. This is generally only possible on insecure protocols so it's likely fine to treat that as a fail.